test/unit/document/document_test.ts

```diff
@@ -1187,11 +1188,27 @@ describe('Document', function () {
     );
   });
 
-  it('Get the value of the counter', function () {
+  it('gets the value of the counter', function () {
     const doc = Document.create<{ counter: Counter }>('test-doc');
     doc.update((root) => {
       root.counter = new Counter(155);
     });
     assert.equal(155, doc.getRoot().counter.getValue());
   });
+
+  it('stringifies any type of rich text attributes to JSON parsable string', function () {
+    const doc = Document.create<{ rich: RichText<Indexable> }>('test-doc');
```

Comment:
I think it would be better to use a type that shows the intention of this work better.

```suggestion
    const doc = Document.create<{
      rich: RichText<{
        bold?: boolean;
        indent?: number;
        italic?: boolean | null;
        color?: string;
      }>
    }>('test-doc');
```